source/_integrations/modbus.markdown

```diff
@@ -1318,6 +1318,13 @@ sensors:
       description: "Suppress values close to zero. If -zero_suppress <= value <= +zero_suppress --> 0. Can be float or integer"
       required: false
       type: float
+    register_size_bytes:
+      description: "Use this field to define a non-standard number of bytes in the request. Only set this value when working when devices that don't fully implement the modbus standard, this values is 2 bytes by the standard.
+      It cannot be used when `data_type: string` where each character is 8 bit.
+      The valid values are `1`, `2`, `4` or `8`"
```

Comment:
The current code also allow the enum value NOT_SET.

Please see the comments in the core PR.